src/tests/Calculator.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Calculator from '../components/Calculator';
+
+test('should render the calculator input correctly', () => {
+  render(<Calculator />);
+  const input = screen.getByRole('textbox');
+  expect(input).toBeInTheDocument();
+});
+test('should render the calculator correctly', () => {
+  render(<Calculator />);
+  const calculator = screen.getByTestId('calc-wrapper');
+  expect(calculator).toMatchSnapshot();
+});
+test('should update the calculator input when the user presses a number button', () => {
+  render(<Calculator />);
+  const input = screen.getByRole('textbox');
+  fireEvent.click(screen.getByText('1'));
+  expect(input.value).toBe('1');
+});
```

Comment:
- [ ] @Team Great work adding tests for the Calculator component, however you have not added tests for all the other components which is part of the project requirements
![image](https://github.com/sagieramos/mathmagicians/assets/61408860/d19cdda3-1b0b-4b37-ac24-c8a84f8c41f8)